classes/controller/AdminController.php

```diff
@@ -436,38 +436,48 @@ public function __construct($forceControllerName = '', $default_theme_name = 'de
+            12 => $this->trans('Module(s) installed successfully.', array(), 'Admin.Modules.Notification'),
+            13 => $this->trans('Module(s) uninstalled successfully.', array(), 'Admin.Modules.Notification'),
+            14 => $this->trans('The translation was successfully copied.', array(), 'Admin.International.Notification'),
+            15 => $this->trans('The translations have been successfully added.', array(), 'Admin.International.Notification'),
+            16 => $this->trans('The module transplanted successfully to the hook.', array(), 'Admin.Modules.Notification'),
+            17 => $this->trans('The module was successfully removed from the hook.', array(), 'Admin.Modules.Notification'),
+            18 => $this->trans('Successful upload.', array(), 'Admin.Notifications.Success'),
+            19 => $this->trans('Duplication was completed successfully.', array(), 'Admin.Notifications.Success'),
+            20 => $this->trans('The translation was added successfully, but the language has not been created.', array(), 'Admin.International.Notification'),
+            21 => $this->trans('Module reset successfully.', array(), 'Admin.Modules.Notification'),
+            22 => $this->trans('Module deleted successfully.', array(), 'Admin.Modules.Notification'),
+            23 => $this->trans('Localization pack imported successfully.', array(), 'Admin.International.Notification'),
+            24 => $this->trans('Localization pack imported successfully.', array(), 'Admin.International.Notification'),
+            25 => $this->trans('The selected images have successfully been moved.', array(), 'Admin.Notifications.Success'),
+            26 => $this->trans('Your cover image selection has been saved.', array(), 'Admin.Notifications.Success'),
+            27 => $this->trans('The image\'s shop association has been modified.', array(), 'Admin.Notifications.Success'),
+            28 => $this->trans('A zone has been assigned to the selection successfully.', array(), 'Admin.Notifications.Success'),
+            29 => $this->trans('Successful upgrade.', array(), 'Admin.Notifications.Success'),
+            30 => $this->trans('A partial refund was successfully created.', array(), 'Admin.OrdersCustomers.Notification'),
+            31 => $this->trans('The discount was successfully generated.', array(), 'Admin.Catalog.Notification'),
+            32 => $this->trans('Successfully signed in to PrestaShop Addons.', array(), 'Admin.Modules.Notification'),
+        );
+
+        $this->_error = array(
+            1 => $this->trans(
+                'The root category of the shop %shop% is not associated with the current shop. You can\'t access this page. Please change the root category of the shop.',
+                array(
+                    '%shop%' => $this->context->shop->name,
+                ),
+                'Admin.Catalog.Feature'
```

Comment:
If this is indeed used in a catalog context, should be "Admin.Catalog.Notification"